### PR TITLE
Rename ContractKit to Solidity SDK

### PR DESCRIPTION
**Pull request by ciaranightingale**
As per Nacho's blessing & request, I have renamed ContactKit to Solidity SDK

### CODE REVIEW

**Comment by jnsdls** [comment]
#### 📝 Changed routes:


| [**/dashboard/contracts**](https://thirdweb-95td0rnol.thirdweb-preview.com/dashboard/contracts) | [**/solidity-sdk**](https://thirdweb-95td0rnol.thirdweb-preview.com/solidity-sdk) |
|:-:|:-:|
|<a href="https://thirdweb-95td0rnol.thirdweb-preview.com/dashboard/contracts">Screenshot of /dashboard/contracts</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/pB98k_uGw5cM60jdsYFO1fJFKmYeewHdllfl8dMaMdqNaEymYKmRnPfdqBrium37V6-474BJZhWD8J_RIMF3fg.png">(view full size)</a> |<a href="https://thirdweb-95td0rnol.thirdweb-preview.com/solidity-sdk">Screenshot of /solidity-sdk</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/QNWxBl1L84tMv7kynNpJsuKvEG6RlzsIwBxzYeOUeG0iqGQRwxj8ftgwbyaqLBacZwepLRjZTqAE2Nr36Z_y8w.png">(view full size)</a> |



| [**/solutions/commerce**](https://thirdweb-95td0rnol.thirdweb-preview.com/solutions/commerce) | [**/solutions/minting**](https://thirdweb-95td0rnol.thirdweb-preview.com/solutions/minting) |
|:-:|:-:|
|<a href="https://thirdweb-95td0rnol.thirdweb-preview.com/solutions/commerce">Screenshot of /solutions/commerce</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/gQizAmzMakikidTQHWT15KhoM2vqpqZza9CMaecbw4YnfPJGIS1REu4sLOmwwYIhRvCqRBDVDoYs0fTMES68_Q.png">(view full size)</a> |<a href="https://thirdweb-95td0rnol.thirdweb-preview.com/solutions/minting">Screenshot of /solutions/minting</a><br /><sup><a href="https://images.weserv.nl?url=https://sjc.microlink.io/g-rsvXnQrhFsM9Ltk0d_IZ5zzCKF0-tfOpFcXoWS75-TGZA2WgpdY-d5By2lrcEZxblZlainz1692dfuFvvZjA.png">(view full size)</a> |







Commit 771bbe4a34e1c3e0dee79f540104a5a4e206933d (https://thirdweb-95td0rnol.thirdweb-preview.com).

File: redirects.js [hunk 1]
```diff
@@ -53,7 +53,12 @@ function redirects() {
...
     {
       source: "/extensions",
-      destination: "/contractkit",
+      destination: "/solidity-sdk",
+      permanent: false,
+    },
+    {
+      source: "/contractkit",
+      destination: "/solidity-sdk",
       permanent: false,
```
Comment (nachoiacovino) [inline redirects.js:62]: let's make this permanent true for SEO purposes

File: pages/solidity-sdk.tsx [hunk 2]
```diff
@@ -25,28 +25,28 @@ const GUIDES = [
...
-    title: "Build a Blockchain Game using ContractKit",
+    title: "Build a Blockchain Game using the Solidity SDK",
     image:
       "https://blog.thirdweb.com/content/images/size/w2000/2022/10/Group-625858--1-.png",
     link: "https://blog.thirdweb.com/guides/build-a-blockchain-game-using-contractkit/",
   },
 ];
 
-const TRACKING_CATEGORY = "contract_kit";
+const TRACKING_CATEGORY = "solidity_sdk";
```
Comment (jnsdls) [inline pages/solidity-sdk.tsx:35]: this will "break" tracking - make sure to update the insights based on this as well

Comment (ciaranightingale) [reply pages/solidity-sdk.tsx:35]: Nacho already warned me - this needs fixing for Release -> publish as well btw!

**Comment by ciaranightingale** [comment]
TODO:
- Add guides tag
- Add links to posthog

File: redirects.js [hunk 3]
```diff
@@ -53,7 +53,19 @@ function redirects() {
...
+    },
+    {
+      source: "/contractkit",
+      destination: "/solidity-sdk",
+      permanent: true,
+    },
+    //  old (deprecated) routes
+    {
+      source:
+        "/:network/(edition|nft-collection|token|pack|nft-drop|signature-drop|edition-drop|token-drop|marketplace|split|vote)/:address",
```
Comment (nachoiacovino) [inline redirects.js:67]:
```suggestion
        "/:network/(edition|nft-collection|token|pack|nft-drop|signature-drop|edition-drop|token-drop|split|vote)/:address",
```

manan changed these to fix a bug, just merge this and it's fine

**Review by nachoiacovino — DISMISSED** [review]
fix the redirects thing and ship it! :D 

Also we need to fix the tag on ghost:

image

**Review by nachoiacovino — APPROVED** [review]
lgtm

**Pull request merged**